rxdiffadapter/src/main/java/com/revolut/rxdiffadapter/RxDiffAdapter.kt

```diff
@@ -47,7 +44,7 @@ open class RxDiffAdapter @Deprecated("Replace with constructor without delegates
     val async: Boolean = false,
     private val autoScrollToTop: Boolean = false,
     private val detectMoves: Boolean = true
-) : AbsRecyclerDelegatesAdapter(delegatesManager) {
+) : DiffAdapter(delegatesManager = delegatesManager, async = async, autoScrollToTop = autoScrollToTop) {
```

Comment:
DiffAdapter now supports both sync/async mode, therefore we don't need to keep rx-based implementation here. 
Let's deprecate this class and make it type alias for DiffAdapter